packages/jest-snapshot/src/State.js

```diff
@@ -108,9 +135,20 @@ export default class SnapshotState {
     }
   }
 
-  match(testName: string, received: any, key?: string) {
+  match({
+    testName,
+    received,
+    key,
+    inlineSnapshot,
+  }: {
+    testName: string,
+    received: any,
+    key?: string,
+    inlineSnapshot?: string,
+  }) {
     this._counters.set(testName, (this._counters.get(testName) || 0) + 1);
     const count = Number(this._counters.get(testName));
+    const isInline = inlineSnapshot !== undefined;
```

Comment:
The changes to this function make it a bit messy and hard to read. Any chance we could make it a tiny bit nicer?